frontend/elements/src/entry/user/email/pages/email-send.ts

```diff
@@ -3,12 +3,10 @@ import "@elements/entry/user/_common/footer/contact";
 import "@elements/entry/user/email/buttons/email-send";
 import "@elements/entry/user/_common/base-page";
 
-const STR_TITLE = "Woohoo! We just sent you an email.";
-const STR_SUBTITLE1 = "You're one step closer to the Jigzi family!";
+const STR_TITLE = "We just sent you an email";
+const STR_SUBTITLE1 = "Please open the email and click on the Verification button. If you cannot see the email in your inbox, check your promotion or spam folders.";
 const STR_SUBTITLE2 =
-    "Please check your inbox and click the verification button.";
-const STR_SUBTITLE3 =
-    "If you don't see it within minutes, check your filters and spam folders.";
+    "I didn’t receive an email. Please send again. (button)";
```

Comment:
I think that the words "Please send again" need to be a button, and "(button)" need to be removed